src/site-provider/SiteProvider.tsx

```diff
@@ -40,11 +41,13 @@ const SiteProvider: React.FC = (props) => {
       >
         <LocaleDataProvider>
           <ScenarioProvider>
-            <FacilityContext.Provider
-              value={{ facility, setFacility, rtData, dispatchRtData }}
-            >
-              {props.children}
-            </FacilityContext.Provider>
+            <FacilitiesProvider>
+              <FacilityContext.Provider
+                value={{ facility, setFacility, rtData, dispatchRtData }}
+              >
+                {props.children}
+              </FacilityContext.Provider>
```

Comment:
I am hoping that that once this new context is done we can get rid of or at least diminish the `FacilityContext` ... at the very least we shouldn't need it to be a single global context anymore, maybe it could sort of be similar to `EpidemicModelContext` where we have multiples on the page and they get their data as a prop rather than having this `setFacility` function and they surround the part of the page that belongs to a single facility. but honestly with all the dispatch functions that exist in `FacilitiesContext` now I'm not sure it will even be necessary anymore. 

I guess either way we will want an easy and stable way to refer to "the current facility" in various places; whether that is a separate context or just a prop that we pass down, I don't have a terribly strong opinion. As this application expands though I have seen the benefits of minimizing the number of places where a tight coupling to a particular context is required. It was a barrier to integrating the curve charts with Rt data, for instance. It's easier in the short term but leaves you with more to unwind when the data source needs to change.

Reply:
Yeah I'm thinking/hoping we could get rid of the FacilityContext altogether. If it becomes too crazy to pass the current facility around as a prop, maybe we could make it a part of the FacilitiesContext state, and add a property `selectedFacility` or something..